### PR TITLE
Drugs

### PR DESCRIPTION
**Pull request by juhis**

### CODE REVIEW

File: pheweb/serve/data_access/db.py [hunk 1]
```diff
@@ -105,6 +115,34 @@ def get_gene_info(self, symbol):
         loc = list(filter( lambda x: x["annotationrelease"]=="109", data["locationhist"]))[0]
         return { "description":data["description"], "summary":data["summary"], "start":data["chrstart"], "stop":loc["chrstop"], "maploc":data["maplocation"]   }
 
+class DrugDao(DrugDB):
+
+    def __init__(self):
+        self.mg = mygene.MyGeneInfo()
```
Comment (Fedja) [inline pheweb/serve/data_access/db.py]: we are already using ensembl for getting gene symbol -> ensembl id . Why introduce additional dependency?

**Comment by juhis** [comment]
Because it seems to be awesome, offers a Python API, gives PharmaGKB data and people are actually using it.. http://mygene.info/apistatus/ <http://mygene.info/apistatus/>

We don’t have a server side implementation of Ensembl lookup but ok I’ll use that

Juha

File: pheweb/serve/static/gene.js [hunk 2]
```diff
@@ -187,14 +219,17 @@ $(function() {
...
-	});
+    $.getJSON("/api/drugs/" + window.gene_symbol)
+	.done(function(data) {
+	    if (data.length > 0) {
+		populate_drugs_streamtable(data)
+	    }
+	})
     $.getJSON("/api/gene_functional_variants/" + window.gene_symbol + "?p=" + window.func_var_report_p_threshold )
 	.done(function(data) {
 	    data.forEach(function(variant) {
```
Comment (Fedja) [inline pheweb/serve/static/gene.js]: empty foreach?

File: setup.py [hunk 3]
```diff
@@ -66,8 +66,9 @@
         'cffi~=1.11',
         'wget~=3.2',
         'gevent~=1.2',
-        'elasticsearch~=6.2.0'
-        'latex~=0.7.0'
+        'elasticsearch~=6.2.0',
+        'latex~=0.7.0',
+        'mygene~=3.0'
```
Comment (Fedja) [inline setup.py]: aaargh

**Comment by juhis** [comment]
:)

File: pheweb/serve/templates/gene_report.tex [hunk 4]
```diff
@@ -62,6 +62,19 @@ \section{ Known GWAS hits in the region}
 ((( var.variant.replace(":"," ") | escape_tex))) \newline ((( var.rsid ))) & ((( var.trait | escape_tex))) & ((( var.risk_allele ))) & ((( var.risk_frq ))) & ((( var.or_beta ))) & ((( var.log_pvalue ))) & (((var.study))) \\
 ((* endfor *))
 \end{longtable}
+
+
+
+\newpage
+
+\section{ Known drugs }
+\begin{longtable}{ >{\RaggedRight}p{3cm}|L{3cm}|c|c|c|c|>{\RaggedRight}p{3cm} }
```
Comment (Fedja) [inline pheweb/serve/templates/gene_report.tex]: add caption as in other tables currently

**Pull request merged**